src/Inceptum.AppServer/AppDiscovery/NuGet/NugetApplicationRepository.cs

```diff
@@ -139,5 +145,10 @@ private void cleanUpInstallFolder(string installPath)
                 File.Delete(packagesConfig);
             }
         }
+
+        private static void ConfigureSupportedSslProtocols()
+        {
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12;

```

Comment:
Maybe you're right, but this is the way nuget team handle issue in future versions:
[NuGet.Clients/NuGet.CommandLine/Program.cs](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.CommandLine/Program.cs#L80)
[NuGet.Core/NuGet.Common/NetworkProtocolUtility.cs](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Common/NetworkProtocolUtility.cs#L15)

I think that's a problem with HttpClient default configuration for work with secure protocols.